src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/jv_jdbc_intro";
+    private static final Properties BD_Properties;
+
+    static {
+        BD_Properties = new Properties();
```

Comment:
Ensure that the variable name `BD_Properties` is corrected to `DB_Properties` for consistency and clarity.